students_proj/src/main/java/com/musala/simple/students/Wrapper.java

```diff
@@ -0,0 +1,13 @@
+package com.musala.simple.students;
+
+/**
+ * This is a helper class used to parse a JSON array of students.
+ * The user of this class can pass it as an argument to the 
+ * {@link com.google.gson.Gson#fromJson(String, Class)} method.
+ * 
+ * @author yoan.petrushinov
+ *
+ */
+public class Wrapper {
```

Comment:
Good idea to use the wrapper! :) Why not renaming it to `StudentWrapper` or something more meaningful? :)